static/js/jobs.js

```
@@ -0,0 +1,271 @@
+            const userLabels = data.user_labels || [];
+
+            // Populate the label picker (action=label)
+            document.getElementById('jobLabelSelect').innerHTML = userLabels.length
+                ? userLabels.map(l => `<option value="${GmailCleaner.UI.escapeHtml(l.id)}">${GmailCleaner.UI.escapeHtml(l.name)}</option>`).join('')
+                : '<option value="">No labels found</option>';
+
+            // Append custom labels to the mailbox dropdown (under a separator)
+            const customLabels = userLabels;
+            const mailboxSel = document.getElementById('jobMailbox');
+            // Remove any previously appended custom labels before re-adding
+            mailboxSel.querySelectorAll('.custom-label-opt').forEach(el => el.remove());
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)